### PR TITLE
updated plugins

### PR DESCRIPTION
**Pull request by danielleadams**
Follow up to https://github.com/heroku/heroku-buildpack-nodejs/pull/731. Adds the updated plugins and some extra test materials.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .anvil
 .DS_Store
 
+node_modules/
+package-lock.json
```
Comment (jmorrell) [inline .gitignore]: Wouldn't adding these cause issues with the test `fixtures`? Or am I missing something?

Comment (danielleadams) [reply .gitignore]: I think I'm missing something... don't we not want these to be committed for fixtures?

Comment (jmorrell) [reply .gitignore]: Most of the time, but there are exceptions in the case that we want to test e.g. how the buildpack reacts when `node_modules` is checked in `test/fixtures/modules-checked-in` or assert how it behaves with and without a lockfile

Comment (danielleadams) [reply .gitignore]: That makes sense, but wouldn't we want to make sure those stay out of git?

**Pull request merged**